earthpy/spatial.py

```diff
@@ -627,6 +627,34 @@ def hillshade(arr, azimuth=30, altitude=30):
     return 255 * (shaded + 1) / 2
 
 
+def crs_check(path):
+    """Get the CRS of a raster file from a file path.
+
+    Parameters
+    ----------
+    path : string
+        Path to the raster file.
+
+    Returns
+    -------
+    crs : crs object
+        The CRS object stored in the raster file.
+    """
+
+    try:
```

Comment:
@nkorinek what happens when you pass a raster with no crs? it may be good to have a message for - there is no crs associated with this file?